File: PROGRAMACION LETICIA/EjersClassesJavaRESPUESTAS/src/ejercicio3/Persona.java
package ejercicio3;

public class Persona {
	private String nombre;
	private int edad;

	public Persona() {
		super();

	}

	public Persona(String nombre, int edad) {
		super();
		this.nombre = nombre;
		this.edad = edad;
	}

	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	public int getEdad() {
		return edad;
	}

	public void setEdad(int edad) {
		this.edad = edad;
	}

	@Override
	public String toString() {
		return "Persona [nombre=" + nombre + ", edad=" + edad + "]";
	}

	public boolean esMayorEdad() {
		boolean resultado = false;
		if (edad >= 18) {
			resultado = true;
		}
		return resultado;
	}

}

File: DAM2/EXAMENES/HLC/utlimo examen/ud05_PP_01_Objetos/ejercicio01/modulo/persona.py
class Persona:
    def __init__(self, nombre, edad):
        self.nombre = nombre
        self.edad= edad
        
    def mostrar_datos(self):
        txt = f"El nombre es {self.nombre} y la edad es {self.edad}"
        return txt
    
    def calcular_edad_futura(self, anios):
        txt = f"Tu edad dentro de {anios} años sera {self.edad+anios}"
        return txt
        
    def get_edad(self):
        return self.edad
    
    def set_edad(self, edad):
        self.edad = edad
    
    def get_nombre(self):
        return self.nombre
    
    def set_nombre(self, nombre):
        self.nombre = nombre
    